### ISSUE TITLE
[Suggestion]Can you add ingame configuration and auto add other mods armor into config?

### ISSUE THREAD

**Issue by JamesChang20022** [defeatedcrow/HeatAndClimateMod#231]

**Comment by defeatedcrow** [comment]
Usually, I accept negotiations and review the content of enhancement requests to decide whether to implement them in the mod.

In this case, despite your request for enhancement, you skipped the consideration of whether or not to implement it, and did not explain the reason for the request or the background circumstances.
You threw an empty description at me, did not negotiate, and urged me to answer with yes no. This technique feels like spam.

In conclusion: There are technical difficulties in implementation, I have no reason to implement it, and I am uncomfortable with such dishonest techniques. This proposal was rejected.

**Comment by JamesChang20022** [comment]
I'm so sorry about not to mention anything about the request, I will try to explain the situation more completely and resend it again later. I apologized any inconvenience caused.
